content/datadog.textile

```diff
@@ -0,0 +1,83 @@
+---
+title: Ably Datadog integration
+meta_description: "Connect Ably and Datadog to monitor messages, channels, and connections in realtime, integrating your Ably metrics with your existing Datadog setup."
+meta_keywords: "Datadog, integrations, metrics, monitoring, analytics, enterprise"
+---
+
+The Ably "Datadog":https://docs.datadoghq.com/integrations/ably/ integration allows "enterprise":/pricing/enterprise users to monitor realtime application metrics like "messages":/messages, "channels":/channels, and connections within Datadog. Data streams every 60 seconds make tracking key performance indicators easier in one place. If you require access, contact "support":https://ably.com/support to enable the integration feature via your "dashboard":https://ably.com/accounts/any.
+
+If you're on the Ably "Pro plan":/pricing/pro, you can enable a limited Datadog-lite integration. "Standard plan":/pricing/standard users may request a 30-day trial to evaluate Datadog-lite before upgrading. This version includes a smaller set of metrics but still provides visibility into your Ably usage. If you need detailed analytics, per-channel breakdowns, custom dashboards, or advanced logging, upgrade to the full Datadog integration available on the Enterprise plan.
+
+|_. Feature |_. Datadog-lite |_. Full Datadog (Enterprise) |
+| Messages | ✅ | ✅ |
+| Connections | ✅ | ✅ |
+| Channels | ✅ | ✅ |
+| Bandwidth | ✅ | ✅ |
+| Message size | ✅ | ✅ |
+| Message direction breakdown (Inbound vs. Outbound) | ❌ | ✅ |
+| Per-Channel metrics | ❌ | ✅ |
+| Usage analytics & trends | ❌ | ✅ |
+| Custom dashboards & alerts | ❌ | ✅ |
+| Advanced logging & debugging | ❌ | ✅ |
+
+
+Ably includes @AppID@ and @AccountID@ tags with each metric to help with data organization. These ID tags enable you to filter, group, and analyze information efficiently, making it more manageable to spot trends and diagnose issues in your Datadog "dashboard":https://docs.datadoghq.com/dashboards/.
+
+To connect Ably and Datadog, you must authorize the integration through Datadog's "OAuth":https://docs.datadoghq.com/developers/integrations/oauth_for_integrations/ flow. This process requires the @api_keys_write@ scope, allowing Ably to push data to your Datadog account. Once the integration is active, Datadog provides a default Ably "dashboard":https://docs.datadoghq.com/integrations/ably/ so that you can monitor key metrics without extra setup.
```

Comment:
Should this be under a 'Configure' or 'Setup' section?

Reply:
I'd also suggest that would be a better place to mention that it needs to be requested to use.